app/assets/stylesheets/components/_filter-section.scss

```diff
@@ -0,0 +1,80 @@
+@import "govuk_publishing_components/individual_component_support";
+@import "govuk_publishing_components/components/mixins/prefixed-transform";
+
+@mixin filter-section-heading-underline {
+  text-decoration: underline;
+  text-underline-offset: 6px;
+  text-decoration-thickness: 4px;
```

Comment:
Is there a reason we want these specific values?

I think `$govuk-link-underline-offset` and `$govuk-link-hover-underline-thickness` might be the way to go for consistency, and because they're more flexible with changing text sizes.

Reply:
Or maybe just use
```scss
@mixin govuk-link-decoration
@mixin govuk-link-hover-decoration
```
... to get everything in one package, and also the `text-decoration-skip-ink` property